packages/amplify-storage-simulator/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "amplify-storage-simulator",
+  "version": "0.0.1",
+  "description": "An S3 simulator to test S3 APIs",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc",
+    "watch": "tsc -w",
+    "start": "node lib/test/test.js"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
We should have the same license as other packages. `Apache-2.0` and author should be `Amazon Web Services`

Reply:
changed the author and license